documentation/writing_tests.md

```diff
@@ -422,6 +424,81 @@ rush add --dev -p @azure/test-utils-recorder
 
 You can read more about the recorder in its readme: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/test-utils/recorder
 
+
+
+## Engineering setup
+
+While writing tests for the Azure SDK works both as a way to verify that our code is correct, and a way to share how to use our code with our customers, we must keep in mind that tests also triggered by automatic systems that generate nightly builds of our packages, that verify that our commits are correct, and that help us have a better level of confidence before releasing anything to the public. 
+
+For our Engineering Systems to pick up our tests appropriately, our packages must be configured according to their guidelines, and our tests must be thought to address their needs. In this section we will go through some of these concepts, and provide links that expand them in detail. We will be covering:
+
+- [Engineering goals](#engineering-goals).
+- [CI configuration files](#ci-configuration-files).
+- [Activating test pipelines](#activating-test-pipelines).
+- [Delivering live tests to our users](#delivering-live-tests-to-our-users).
+
+### Engineering goals
+
+Though the tests for the Azure SDK for JavaScript and TypeScript must target live resources, we should make sure they only do so when necessary. For this purpose, we must keep in mind the following guidelines:
+
+- Tests should not be flaky. Tests should pass regardless of who's executing it, when it is running, and how many times does it run.
+- While writing tests, use static resources for your tests when possible to avoid repeatedly incurring delays for setting up and tearing down resources. Once tests are pushed into the repository, the pipelines will create these resources for us. We'll examine how to set this up in the following sections.
```

Comment:
Live tests should use dynamic resources whenever possible (as opposed to static resources). I updated our language over at https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/51/Testing-Guidelines to reflect that.